data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			action
+				"reputation: Wanderer" = 0
+			label neutral
+			`	"Scout all of their systems for us, and bring back any information you may gather."`
+			choice
+				`	"I guess I'll try to land on their planets."`
+					accept
+				`	"I am not interested in being able to communicate with them."`
+			`	"Well if you want to work with us, you'll have to set that preferance aside. And don't forget: the translator is for communicating with us as well. Some among the most conversative of us despise speaking in your language. This will be a token of good faith on your part."`
+			choice
+				`	"I guess I'll try to land on their planets."`
+					accept
+				`	"I will not work for you anymore then."`
+					decline
+	waypoint "Chy'chra"
+	waypoint "Pik'ro'iyak"
+	waypoint "Ik'kara'ka"
+	waypoint "Chirr'ay'akai"
+	waypoint "Sich'ka'ara"
+	waypoint "Kiro'ku"
+	waypoint "Ka'pru"
+	waypoint "Ap'arak"
+	waypoint "Ka'ch'chrai"
+	waypoint "Ek'kek'ru"
+	waypoint "Si'yak'ku"
+	waypoint "Iyech'yek"
+	waypoint "Kiru'kichi"
+	waypoint "Es'sprak'ai"
+
+	on waypoint
+		dialog `You have scouted all of the Wanderer's systems. If you can speak Hai already, it's time to come back to <destination> for your reward. Otherwise, go to <stopovers>.`
+	to complete
+		has "language: Wanderer"
+	on complete
+		log `The Unfettered asked me to be able to speak like a True Hai, because that is the language in which I should make demands to the Unfettered later.`
+		log "Wanderer" `The Wanderers are a birdlike species that inhabits the old territories of the Unfettered Hai.`
+		payment 1000000
+		dialog `You land and find the herald you spoke to earlier. "We now know we can speak to you in the True language of the Hai, finally. And you can speak to us in Hai, through the translator. Most of us will be willing to continue communicating in your language, but expect to use the translator to speak and understand our Chiefs. They'll probably act like it's nothing, but trust me, they care. Here's <payment> for the intel."`
+
+
+
+mission "Unfettered: Bring Detainer"
+	landing
+	name `Recover a detainer`
+	description `He Yi has asked you to earn a translation device by bringing back a powerful alien weapon, said to be able to neutralize the engines of its target. You should probably look around Kor Sestor space.`
+	source "Warfeed"
+	to offer
+		has "Unfettered: Know Your Enemy: active"
+		has "First Contact: Wanderer: offered"
+		not "language: Wanderer"
+	on offer
+		conversation
+			`You go find He Yi in his war room, who does not seem too surprised to see you. "I expected you to come here. You must want one of us to act as 'diplomat' with the aliens to the north."`
```

Comment:
`You must want one of us to act as 'diplomat' with the aliens to the north.` This is wrong - the herald simply told us that's what's happening. We didn't want it.

Reply:
I added for completing the mission we asked of you, or smth like that